File: baekjoon/10773/10773.cpp
#include<iostream>
#include<stack>
using namespace std;

int main(){
    stack<int> st;
    int n;cin>>n;

    int num;
    while(n--){
        cin>>num;
        if(num == 0)
            st.pop();
        else
            st.push(num);

    }

    int sum=0;
    if(st.size() == 0) sum=0;
    else{
        while(!st.empty()){
            sum+=st.top();
            st.pop();
        }
    }
    cout<<sum<<'\n';

    return 0;
}

File: baekjoon/10773/10773.py
import sys
input = sys.stdin.readline

stack = []
k = int(input())
for i in range(k):
    num = int(input())
    if(num == 0):
        stack.pop()
    else:
        stack.append(num)

sum = 0
if(len(stack) != 0):
    for i in range(len(stack)):
        sum += stack[i]
print(sum)
